### PR TITLE
Temporarily disable two MLflow tests for MacOS with Python 3.9 and 3.10

### PR DESCRIPTION
**Pull request by strickvl**
## Describe changes
I implemented/fixed _ to achieve _.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).
- [ ] If my changes require changes to the dashboard, these changes are communicated/requested.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

## Summary by CodeRabbit

- **Tests**
  - Temporarily disabled specific integration test scripts from running on macOS with Python 3.9 or 3.10 to maintain compatibility.

### CODE REVIEW

**Comment by strickvl** [comment]
@coderabbitai review

File: tests/integration/examples/test_mlflow_registry.py [hunk 1]
```diff
@@ -12,6 +12,15 @@
...
 
+import platform
+import sys
+
+# TEMPORARILY DISABLED
+if platform.system() == "Darwin" and sys.version_info[:2] in [(3, 9), (3, 10)]:
+    print(
+        "This script is not intended to run on macOS with Python 3.9 or 3.10."
+    )
+    sys.exit()
```
[the remark this answers is not in the document]
Comment (strickvl) [reply tests/integration/examples/test_mlflow_registry.py:23]: but this is outside a pytest test. How does it work? Would it just skip adding any / all tests inside this file when we hit pytest.skip()?

**Comment by strickvl** [comment]
@coderabbitai review

**Review by bcdurak — COMMENTED** [review]
@strickvl Do we have an update on whether the MLflow issue is resolved or not?

Otherwise, the changes look good.

**Comment by strickvl** [comment]
@bcdurak it is not resolved :) we are the team that has to resolve it, but I'd disable these two temporarily here until we get the fix. @avishniakov was working on debugging this in #2206 before he left for vacation. We'll pick this back up next week I think.

**Review by bcdurak — APPROVED** [review]

**Pull request merged**